app/models/world_location_type.rb

```diff
@@ -1,24 +0,0 @@
-class WorldLocationType
-  include ActiveRecordLikeInterface
-
-  attr_accessor :id, :name, :sort_order, :key
-
-  def slug
-    name.downcase.gsub(/[^a-z]+/, "-")
```

Comment:
Leaving half a thought here before I have a bit more time to delve into this - was this slug used anywhere except in the find_by_slug below (and was that used anywhere)? Just want to make sure that this isn't somehow used to generate the slug of the overall document (I think not but haven't fully looked into this yet)

Reply:
There's a `slug` database field in the `world_locations` table, so Rails automatically generates a `slug` method for these objects.